db/influx.go

```diff
@@ -172,6 +175,68 @@ func (i *influxDB) TodaysCounts() (map[string]api.RelayCounts, error) {
 	return counts, nil
 }
 
+func (i *influxDB) TodaysLatencies() (map[string][]api.Latency, error) {
+	client := influxdb2.NewClient(i.Options.URL, i.Options.Token)
+	queryAPI := client.QueryAPI(i.Options.Org)
+
+	latencies := make(map[string][]api.Latency)
+	// TODO: send queries in parallel
+	query := fmt.Sprintf("from(bucket: %q)", i.Options.CurrentBucket) +
+		fmt.Sprintf(" |> range(start: %s)", startOfDay(time.Now()).Format(time.RFC3339)) +
```

Comment:
I know this issue was already there in the code, but there must be a more readable way to deal with the query and its parameters. One potential solution is use of `text/template` package to make the query readable while allowing variable substitution.

Reply:
What would you think of this approach: 
![image](https://user-images.githubusercontent.com/47662958/196822228-0199e689-a380-4fbd-ab9c-c32a1d50aaae.png)